quick-github-guide.md

```diff
@@ -0,0 +1,163 @@
+
+1. Make sure **Allow edits from maintainers** remains checked.
+
+1. Click **Create pull request** again. You have now sent a request to the repository
+  maintainers to review your change.
+
+1. Check the online preview of your changes:
+
+      * Wait for the automated PR workflow to do some checks. When it's ready,
+        you should see a comment like this: **deploy/netlify — Deploy preview ready!**
+      * Click **Details** to the right of "Deploy preview ready" to see a preview
+        of your updates.
+
+## Using the command line
+
+Here's a quick guide to a fairly standard GitHub workflow using git on the command line:
+
+1. Fork the kubeflow/website repo:
+
+    * Go to the [kubeflow/website repo](https://github.com/kubeflow/website) on GitHub.
+    * Click **Fork** to make your own copy of the repo. GitHub creates a copy
+      at `https://github.com/<your-github-username>/website`.
+
+1. Open a command window on your local machine.
+
+1. Clone your forked repo, to copy the files down to your local machine.
+  This example creates a directory called `kubeflow` and uses SSH cloning to
+  download the files:
+
+    ```
+    mkdir kubeflow
+    cd kubeflow/
+    git clone git@github.com:<your-github-username>/website.git
+    cd website/
+    ```
+
+1. Add the upstream repo as a git remote repo:
+
+    ```
+    git remote add upstream https://github.com/kubeflow/website.git
+    ```
+
+1. Check your remotes:
+
+    ```
+    git remote -vv
+    ```
+
+    You should have 2 remote repos:
+
+      -  `origin` - points to your own fork of the repo on gitHub -
+         that is, the one you cloned my local repo from.
+      -  `upstream` - points to the actual repo on gitHub.
+
+1. Create a branch. In this example, replace `doc-updates` with any branch name
+  you like. Choose a branch name that helps you recognise the updates you plan
+  to make in that branch:
+
+    ```
+    git checkout -b doc-updates
+    ```
+
+1. Add and edit the files as you like. The doc pages are in the
+  `/website/content/docs/` directory.
+
+1. Run `git status` at any time, to check the status of your local files.
+  Git tells you which files need adding or committing to your local repo.
+
+1. Commit your updated files to your local git repo. Example commit:
+
+    ```
+    git commit -a -m "Fixed some doc errors."
+    ```
+
+    Or:
+
+    ```
+    git add add-this-doc.md
+    git commit -a -m "Added a shiny new doc."
+    ```
+
+1. Push from your branch (for example, `doc-updates`) to **the relevant branch
+  on your fork on GitHub:**
+
+    ```
+    git checkout doc-updates
+    git push origin doc-updates
+    ```
+
+1. When you're ready to start the review process, create a pull request (PR)
+  **in the branch** on **your fork** on the GitHub UI, based on the above push.
+  The PR is auto-sent to the upstream repo - that is, the one you forked from.
+
+1. If you need to make changes to the files in your PR, continue making them
```

Comment:
Suggestion: shorter version

If you need to change the files in your PR,

Reply:
Weird, these suggestions didn't appear in the list of review comments the first time round. I'm addressing them now and will do another commit and push.

Done